### DISCUSSION TITLE
stackTrace

### DISCUSSION THREAD

**Discussion by jnxyatmjx** [chenshuo/muduo#666] [General]
DELETE FROM HERE TO LINE 18 BEFORE OPEN AN ISSUE.

GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, use GitHub discussions:

https://github.com/chenshuo/muduo/discussions

When filing an issue of muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

If you can't compile muduo, make sure you install `cmake` and `boost` from the
official package repository, e.g. `apt` or `yum`, before opening a bug.
Don't open a bug if you installed boost from a third-party source or
downloaded it by yourself, and couldn't compile muduo, thank you.

Also specify the exact environment where the issue occurs:

## Version of cmake, gcc and boost? (If not from distro.)
请教下，stackTrace 这个函数输出内容，怎样定位到具体的 行数呢

**Comment by ggandycong** [comment]
刘德聪，已经收到你的邮件，请你放心
祝你生活愉快，笑口常开

**Comment by jnxyatmjx** [comment]
> 刘德聪，已经收到你的邮件，请你放心 祝你生活愉快，笑口常开

你怎么知道我名字的？
